shuttle-cron/Cargo.toml

```diff
@@ -8,8 +8,8 @@ publish = false
 apalis = { version = "0.4.9", features = ["cron", "postgres", "extensions", "retry"] }
 chrono = { version = "0.4.32", features = ["clock", "serde"] }
 serde = { version = "1.0.195", features = ["derive"] }
-shuttle-runtime = "0.37.0"
-shuttle-shared-db = { version = "0.37.0", features = ["postgres"] }
-sqlx = { version = "0.7.3", features = ["runtime-tokio-native-tls", "postgres"] }
+shuttle-runtime = "0.46.0"
+shuttle-shared-db = { version = "0.46.0", features = ["postgres"] }
+sqlx = { version = "0.7.1", features = ["runtime-tokio-native-tls", "postgres"] }
```

Comment:
This has to stay on a lower version of runtime+shared-db for a while since the cargo+sqlx feature bug pulls in 2 versions of libsqlite. Seems like it's impossible to support both versions (like for axum) due to it. cc @oddgrd